USER_MANUAL.md

```diff
@@ -114,5 +114,16 @@ Actors used in gyms are in `Contetnt\Actors`: add any new Actors to this directo
 * Demonstrates that ClientWorkers and UnrealWorker read the correct value for the "test" worker flag, when both types have a "test" flag with different value.
 * Validation:
   1. Use workerflags_testgym_config launch config to run multiplayer through the editor, for every worker running, different values based on their worker type should get printed/logged.
+
+##### Server to server RPC gym
+* Tests server to server RPCs are working. Sends RPCs to actors owned by different servers and visualises when they are received.
+* Contains a set of cubes placed in four quadrants of the level. The cubes change colour to indicate which worker the owning worker
```

Comment:
Might want to say how the RPCs are sent (i.e. random at an interval). Then maybe reverse this sentence to say when an rpc is received the target cube will change colour to indicate the sending worker.

Reply:
done